spec/models/article_spec.rb

```diff
@@ -484,23 +484,33 @@ def build_and_validate_article(*args)
     end
 
     it "sets published_at from a valid frontmatter date" do
-      date = (Date.current - 5.days).strftime("%d/%m/%Y")
+      date = (Date.current + 5.days).strftime("%d/%m/%Y")
```

Comment:
Reading note:

There is a spec later on that was `it "rejects future dates set from frontmatter"`; however that is no longer true